webpack.dev.config.babel.js

```diff
@@ -69,6 +69,11 @@ export default Object.assign({}, webpackConfig, {
   module: {
     rules: getRules({ babelQuery: BABEL_QUERY, bundleStylesWithJs: true }),
   },
+  node: {
```

Comment:
You don't need this because all configs [extend the prod config](https://github.com/willdurand/addons-frontend/blob/fd31ac940d0fc509509d648f96117293ffc8f29a/webpack.dev.config.babel.js#L56).

Reply:
right. I forgot to remove it after having tried a few things...